x/ccv/provider/keeper/throttle.go

```diff
@@ -195,7 +195,12 @@ func (k Keeper) GetSlashMeterAllowance(ctx sdktypes.Context) sdktypes.Int {
 func (k Keeper) QueueGlobalSlashEntry(ctx sdktypes.Context, entry providertypes.GlobalSlashEntry) {
 	store := ctx.KVStore(k.storeKey)
 	key := providertypes.GlobalSlashEntryKey(entry)
-	store.Set(key, entry.ProviderValConsAddr)
+	bz, err := entry.ProviderValConsAddr.Marshal()
+	if err != nil {
+		// This should never happen, since the provider val cons addr should be a valid sdk address
```

Comment:
We again assume the provider cons address has been properly validated by now. This is where it's obtained https://github.com/cosmos/interchain-security/blob/38a1540972d3cd1dda7285c5a602b8706840270f/x/ccv/provider/keeper/relay.go#L316